www-client/uget/uget-2.2.1.ebuild

```diff
@@ -28,13 +28,14 @@ RDEPEND="
 	appindicator? ( dev-libs/libappindicator:3 )
 	gstreamer? ( media-libs/gstreamer:1.0 )
 	libnotify? ( x11-libs/libnotify )
-	openssl? ( dev-libs/openssl:0 )
-	"
-DEPEND="${RDEPEND}
+	openssl? ( dev-libs/openssl:0 )"
```

Comment:
check if this links, and if it does, add `:0=`

Reply:
Does not link against openssl